src/wkwebview/mod.rs

```diff
@@ -71,6 +72,8 @@ const ACCEPT_FIRST_MOUSE: &str = "accept_first_mouse";
 
 const NS_JSON_WRITING_FRAGMENTS_ALLOWED: u64 = 4;
 
+static WEBVIEW_IDS: Lazy<Mutex<Vec<usize>>> = Lazy::new(Default::default);
```

Comment:
I'm okay with the global variable approach. Just slightly concerned that the ID will conflict when removing and adding a webview immediately. The ID may be the same.

I remember that muda has a similar way of dealing with the ID in a global variable: https://github.com/tauri-apps/muda/blob/dev/src/util.rs

Maybe using a Counter + HashSet will be better?

Reply:
i don't think this should be possible but better safe than sorry, i'll copy the counter.